.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4

```

Comment:
These are not the correct settings for all languages, e.g. we use `indent_size = 2` for TypeScript.
Let's move this file to the `src/python` directory.

Reply:
I can add ts files here too.

Reply:
If you want to add a global `.editorconfig`, it should cover all languages used in the repository (TypeScript is just an example, Ruby also uses 2 spaces). So you'll have to check the code styles of each runtime. If you don't want to do this, simply move this file to `src/python`.